src/tinyformat.h

```diff
@@ -167,7 +167,7 @@ namespace tinyformat {
 class format_error: public std::runtime_error
 {
 public:
-    format_error(const std::string &what): std::runtime_error(what) {
+    format_error(const std::string &fmtwhat): std::runtime_error(fmtwhat) {
```

Comment:
No. silencing an obnoxious shadowing warning that was making it impossible to see where other issues were.